MattTuring/pallet-picker-backend
# Pallet table

### What's this PR do?

This PR adds migrations and seeds for project table to development branch and contains next changes: 

- Schema for 'pallets' table
- Seed simple data to 'pallets' table in DB

### Where should the reviewer start?

PR contains explanation comments and reviewer can start from the top of comments.

### How should this be manually tested?

- `git fetch`
-  `git checkout pallet-table`
- to be sure that you have database for this project in Postico
- `knex migrate:latest`
- refresh database
- check that table was created with correct columns
- `knex seed:run`
- the foreign key id in this setup would be sent via post request, make sure the id will be real when testing. Ex. if you have id 1 & 2 in the projects table on the next seed run they will be 3 & 4. Make the pallet data match 3 or 4 before running the seed
- check that table was filled
